app/scripts/migrations/107.test.ts

```diff
@@ -0,0 +1,157 @@
+import { migrate } from './107';
+
+describe('migration #107', () => {
```

Comment:
There is one notable test case missing here, which is if the CachedBalancesController state exists, but is an empty object, and if CachedBalancesController state is `( cachedBalances: {} )`